src/error-index-generator/main.rs

```diff
@@ -73,7 +73,7 @@ r##"<!DOCTYPE html>
 
     try!(write!(&mut output_file, "<h1>Rust Compiler Error Index</h1>\n"));
 
-    for (err_code, info) in err_map.iter() {
+    for (err_code, info) in err_map {
```

Comment:
This kind of thing is extra interesting, because this _looks_ like a change in semantics, but since `err_map` is a reference, it's not.